init.py

```diff
@@ -1,7 +1,9 @@
 from chatbot import setup
 from chatbot import message_to_bot
-clf, learn_response = setup()
+clf, learn_response = setup()    
 while(True):
-    received_message = input("You: ")
-    send_message, learn_response = message_to_bot(received_message, clf, learn_response)
-    print("MapBot:  " + send_message)
+	received_message = input("You: ")
+	send_message, learn_response = message_to_bot(received_message,clf,learn_response)
+	print("MapBot: "+send_message)
+	if received_message.lower() == "bye" or received_message.lower() == "bye." or received_message.lower() == "bye!":															#exit loop
```

Comment:
`chatbot.py` already handles 'bye'. Use the same to avoid redundant code here. Cleanly exiting is not implemented there, which could be added in `init.py`.